lib/multiSemanticRelease.js

```diff
@@ -180,10 +239,29 @@ async function releasePackage(pkg, createInlinePlugin, multiContext) {
 
 	// Call semanticRelease() on the directory and save result to pkg.
 	// Don't need to log out errors as semantic-release already does that.
-	pkg.result = await semanticRelease(options, {
+	pkg.result = semanticRelease(options, {
 		cwd: dir,
 		env,
-		stdout: new RescopedStream(stdout, name),
-		stderr: new RescopedStream(stderr, name),
-	});
+		stdout: new RescopedStream(pkg.stdout, name),
+		stderr: new RescopedStream(pkg.stderr, name),
+	}).then(
+		(result) => {
+			console.log({
+				lastRelease: result.lastRelease,
+				nextRelease: result.nextRelease,
+				releases: result.releases,
+			});
+
+			pkg.isPending = false;
```

Comment:
`isPending` was added when I was modifying the original algorithm of the synchronizer, so can safely be ignored now — I was using this to determine the `todo()`'s instead of the presence or absence of the `pkg.result` property